ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml

```diff
@@ -129,6 +137,27 @@ tests:
       GATHER_CAPI_LOGS: "true"
       TEST_SUITE: full
     workflow: assisted-baremetal
+- as: e2e-assisted-operator-capi-periodic
```

Comment:
Before this job, can you add the presubmit equivalent?
So that you can test any periodic test in PRs as well

Maybe even make it ``alwaysRun: false`` for now, to be totally optional in running